scaleway/resource_vpc_gateway_network_test.go

```diff
@@ -15,7 +15,7 @@ func init() {
 		Name: "scaleway_gateway_network",
 		F:    testSweepVPCGatewayNetwork,
 		// test depends upon PrivateNetwork, PublicGateway. Please add new resources for testing purpose.
-		Dependencies: []string{"scaleway_vpc", "scaleway_vpc_gateway_ip"},
```

Comment:
Why is this change needed?

Reply:
Absolutely the call of the sweep vpc was enough to remove the resource vpc. This was causing an error `already called`